recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -16,5 +16,8 @@ conda install -n root --yes --quiet jinja2 conda-build anaconda-client
 conda install -n root --yes --quiet vs2008_express_vc_python_patch
 call setup_x64
 
+:: Set the conda-build working directory to a smaller path
+set "CONDA_BLD_PATH=C:\\bld\\"
```

Comment:
Not really, @msarahan didn't like the idea of putting it to `C:\\`. That's why I put it into `C:\\bld\\`. Shall I change this `C:\\`?